src/hooks/useNetworkMotionStatesAllColonies.ts

```diff
@@ -0,0 +1,189 @@
+import {
```

Comment:
I would really appreciate for review this code, it kinda duplication for https://github.com/JoinColony/colonyCDapp/blob/master/src/hooks/useNetworkMotionStates.ts 
But I tried to do it for all user colonies, not only for the current one

Reply:
Good shout with a separate hook for all colonies. I think it can be simplified a bit further to roughly follow this flow:
- Get all installed extensions for joined colonies
- Select all voting reputation addresses (directly from the query data, you shouldn't need any mapping)
- Get network states for each of the motions
- Create a map between motion database IDs and states so that a flat map can be used